README.md

```diff
@@ -21,6 +21,34 @@ Official guideline resources:
 - reddit: https://www.reddit.com/about/alien/
 - http://findguidelin.es
 
+## Using the SocialLogo Component in your project:
+
+Note that this component requires [react](https://www.npmjs.com/package/react) and [classnames](https://www.npmjs.com/package/classnames)
```

Comment:
`classnames` should be declared as a `peerDependencies` then. Though... it would be nice if we removed this prerequisite altogether. Surely we can generate the equivalent code ourselves without too much trouble.